static-analysis/teammates-tslint.yml

```diff
@@ -150,7 +150,6 @@ rules:
   no-unsafe-finally: true
   no-unused-expression: true
   no-unused-variable: false # Use compiler's built-in check instead
-  no-use-before-declare: true
```

Comment:
Are we removing `no-use-before-declare` and `arrow-call-signature`?

Reply:
`no-use-before-declare` was deprecated. `arrow-call-signature` returns too many violations and is also not a very value-adding rule to add after the new TSLint version update.